src/Controls/src/Core/LegacyLayouts/RelativeLayout.cs

```diff
@@ -11,6 +11,7 @@
 namespace Microsoft.Maui.Controls.Compatibility
 {
 	[ContentProperty(nameof(Children))]
+	[Obsolete("Use Microsoft.Maui.Controls.RelativeLayout instead. For more information, see https://learn.microsoft.com/dotnet/maui/migration/layouts")]
```

Comment:
Updates this text to just indicate that we are obsoleting RelativeLayout. `Microsoft.Maui.Controls.RelativeLayout` currently doesn't exist so we can just point them to the documentation you've referenced